app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+  <%= comment.body %>
+  <% if comment.user %>
```

Comment:
The `<% if comment.user %>` check shouldn't be necessary, all comments ought to have a user. If you remove line 3 and 5 and get an error we ought to debug it.